### PR TITLE
Add drbd.conf

### PR DESCRIPTION
**Pull request by Wescoeur**

### CODE REVIEW

File: SPECS/xcp-ng-release-linstor.spec [hunk 1]
```diff
@@ -19,8 +21,12 @@ sed -i "s/\# \(global_filter\)[[:space:]]*=.*/\1 = [ \"r|^\/dev\/drbd.*|\" ]/g"
 
 %files
 %config(noreplace) %{_sysconfdir}/yum.repos.d/xcp-ng-linstor.repo
+%config(noreplace) %{_sysconfdir}/modprobe.d/drbd.conf
 
 %changelog
+* Mon Mar 13 2023 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.2-1
+- Add drbd.conf to prevent idle TCP connections from freezing.
```
Comment (stormi) [inline SPECS/xcp-ng-release-linstor.spec]: Maybe mention that it's a modprobe configuration, here and in the commit message too.

File: SPECS/xcp-ng-release-linstor.spec [hunk 2]
```diff
@@ -19,8 +21,12 @@ sed -i "s/\# \(global_filter\)[[:space:]]*=.*/\1 = [ \"r|^\/dev\/drbd.*|\" ]/g"
 
 %files
 %config(noreplace) %{_sysconfdir}/yum.repos.d/xcp-ng-linstor.repo
+%config(noreplace) %{_sysconfdir}/modprobe.d/drbd.conf
```
Comment (stormi) [inline SPECS/xcp-ng-release-linstor.spec]: `%config(noreplace)` means if users modify the configuration future changes from the RPM regarding this file will not be applied (a .rpmnew file will be created instead). Is this what you want, or do you prefer to overwrite the file if you need to bring modifications to it in the future? If you chose to do so, a warning on top of the file will be appropriate.

**Review by benjamreis — APPROVED** [review]

File: SPECS/xcp-ng-release-linstor.spec (hunk 2 above)
Comment (Wescoeur) [reply SPECS/xcp-ng-release-linstor.spec]: I don't want to overwrite the file, especially if a user needs particular settings. It doesn't seem impossible to me.

Comment (stormi) [reply SPECS/xcp-ng-release-linstor.spec]: Couldn't we offer cascading configuration files instead, Like, our defaults in `00-drbd.conf` and user settings or overrides in another file?

Comment (Wescoeur) [reply SPECS/xcp-ng-release-linstor.spec]: We can't, it's a kernel module config, we can just use this file.

**Review by stormi — APPROVED** [review]

File: SPECS/xcp-ng-release-linstor.spec (hunk 2 above)
Comment (stormi) [reply SPECS/xcp-ng-release-linstor.spec]: Solved by putting the file in /usr/lib/modprobe.d/ which is the place for default distro configuration, and leaves /etc to the user.

**Pull request merged**